postgres/cdk/lib/giant.ts

```diff
@@ -53,8 +50,9 @@ export class Giant extends GuStack {
 				}),
 			},
 			engine: DatabaseInstanceEngine.postgres({
-				version: PostgresEngineVersion.VER_13,
+				version: PostgresEngineVersion.VER_15
 			}),
+            allowMajorVersionUpgrade: true,
```

Comment:
Just for my own understanding, what would happen if we don't use this field? `allowMajorVersionUpgrade`

Reply:
The upgrade would be blocked by RDS - it's like an extra safety feature. Once this is merged and the DB has been upgraded on PROD i think we should remove this line again so we don't accidentally upgrade!